tests/validation/test_entropy.py

```diff
@@ -40,6 +41,34 @@ def test_discrete_entropy_two_values(self):
             diff_val_h,
             msg='Expected entropy in x ~ Ber(0.5)')
 
+    def test_entropy_cont_disc_heuristics(self):
+        arange_disc_arr = asarray2d(np.arange(50))
+        arange_cont_arr = asarray2d(np.arange(50) + 0.5)
+
+        disc_h = estimate_entropy(arange_disc_arr)
+        cont_h = estimate_entropy(arange_cont_arr)
+        self.assertNotEqual(
+            disc_h,
+            cont_h,
+            msg='Expected cont, disc columns to be handled differently')
```

Comment:
handled differently, for sure, but does that require their entropy is not equal?

Reply:
I'll have to think about this. Taken from the sklearn docs - "Also note, that treating a continuous variable as discrete and vice versa will usually give incorrect results, so be attentive about that."